pkgs/by-name/vd/vdhcoapp/package.nix

```
@@ -0,0 +1,86 @@
+{ lib
+, fetchFromGitHub
+, buildNpmPackage
+, nodejs
+, ffmpeg
+, callPackage
+, filepicker ? callPackage ./filepicker.nix {}
+, substituteAll
+, makeWrapper
+, toml2json
+, jq
+}:
+
+buildNpmPackage rec {
+  pname = "vdhcoapp";
+  version = "2.0.19";
+  srchash = "sha256-8xeZvqpRq71aShVogiwlVD3gQoPGseNOmz5E3KbsZxU=";
+
+  src = fetchFromGitHub {
+    owner = "aclap-dev";
+    repo = "vdhcoapp";
+    rev = "v${version}";
+    hash = srchash;
```

```suggestion

  src = fetchFromGitHub {
    owner = "aclap-dev";
    repo = "vdhcoapp";
    rev = "v${version}";
    hash = "sha256-8xeZvqpRq71aShVogiwlVD3gQoPGseNOmz5E3KbsZxU=";
```
if you need to mark that special for an updater script or so, please use an comment instead